sdk/python/feast/registry.py

```diff
@@ -1,1871 +1,12 @@
-# Copyright 2019 The Feast Authors
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     https://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-import abc
-import json
-import logging
-import uuid
-from abc import abstractmethod
-from collections import defaultdict
-from datetime import datetime, timedelta
-from enum import Enum
-from pathlib import Path
-from threading import Lock
-from typing import Any, Dict, List, Optional
-from urllib.parse import urlparse
+import warnings
 
-from google.protobuf.internal.containers import RepeatedCompositeFieldContainer
-from google.protobuf.json_format import MessageToJson
-from proto import Message
+from feast.infra.registry.registry import BaseRegistry, Registry  # noqa
```

Comment:
might be able to avoid this by using https://github.com/feast-dev/feast/blob/master/sdk/python/feast/__init__.py#L23-L23 ?

Reply:
hmm @adchia not sure that works? the new `registry.py` containing `BaseRegistry` is on a different level

also the `noqa` thing is to avoid errors since `BaseRegistry` and `Registry` are imported and not used

Reply:
not quite sure what you mean? BaseRegistry and Registry are in different files, yes, but you can make feast alias them into feast.Registry or feast.BaseRegistry

we do this already for the other offline/online stores which are similarly on a different level (e.g. you can do

```python
from .infra.registry.base_registry import BaseRegistry
from .infra.registry.registry import Registry
```

and add to the __all__ variable
) and then remove this file

Reply:
after discussion, decision was to just delete `registry.py` since there's no good reason for a user to be doing `from feast.registry import Registry` right now